src/pages/[post].tsx

```
@@ -204,6 +204,13 @@ export default function PostPage({
                 </Link>
               )}
               <TagList tags={tags} resourceSlug={post.fields.slug ?? ''} />
+              <CopyAsPromptButton
```

ESLint

[error] 'CopyAsPromptButton' is not defined.

(react/jsx-no-undef)